notebooks/official/tensorboard/vertex_tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -0,0 +1,900 @@
+{
```

Comment:
<p>Use the standardized way to set a service account otherwise the service account won't be injected correctly in cloud build tests.</p><p> </p><pre class="ql-syntax" spellcheck="false">SERVICE_ACCOUNT = "[your-service-account]"  # @param {type:"string"}
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>done</p>